tests/test_base_plugin.py

```
@@ -1643,6 +1643,59 @@ def test_parse_mesh_bang_command_non_string_or_empty_commands(self):
         self.assertIsNone(plugin.parse_mesh_bang_command(12345, ("weather",)))
         self.assertIsNone(plugin.parse_mesh_bang_command("!weather", ()))
 
+    @patch("mmrelay.matrix_utils.connect_matrix")
+    def test_send_matrix_reaction_success(self, mock_connect):
+        """Test send_matrix_reaction sends reaction when client available (lines 768-785)."""
+        plugin = MockPlugin()
+        plugin.logger = MagicMock()
+        mock_client = AsyncMock()
+        mock_connect.return_value = mock_client
+
+        async def run_test():
```

Ruff (0.15.10)

[warning] Missing return type annotation for private function `run_test`

Add return type annotation: `None`

(ANN202)